### PR TITLE
Runtime expression compatibility analysis.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/main/scala/cromwell/engine/CromwellActor.scala [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 import scala.language.postfixOps
 
 trait CromwellActor {
-  implicit val timeout = Timeout(5 seconds)
+  protected implicit val timeout = Timeout(5 seconds)
+
+  protected val log: LoggingAdapter
+
+  /**
+   * Hack to log just an error message so Akka TestKit can pick up on it, and then again to log
```
Comment (geoffjentry) [inline src/main/scala/cromwell/engine/CromwellActor.scala]: I long for a world where we didn't have to go through crazy hijinks just to make our testing overlords compliant and happy.

**Comment by geoffjentry** [comment]
:+1: from me but i'll wait a little bit to see if anyone else pipes up

File: src/main/scala/cromwell/engine/CromwellActor.scala (hunk 1 above)
Comment (mcovarr) [inline src/main/scala/cromwell/engine/CromwellActor.scala]: This is kind of horrible.  I'd like to spend some cycles today digging into this as I'm currently under the budgeted points.  :smile:

**Comment by scottfrazer** [comment]
Rebase?

**Pull request merged**